src/components/tool/guide/EresearchJob/EresearchJob.js

```diff
@@ -121,6 +122,60 @@ const getConfigGroups = (config, onConfigChange = () => {}) => {
         selected: (config) => config?.nodes > 0,
       };
     },
+    isArrayJob: () => {
+      const arrayJobEnabled = [
+        ["Yes", "Sets job's array attribute to True"],
+        ["No", "Sets job's array attribute to False"],
+      ];
+      return {
+        element: (key, selected) => (
+          <ConfigGroup
+            key={key}
+            title="Array Job"
+            description="Make this job an array job"
+            type="picker"
+            selected={selected}
+            inputProps={{
+              choices: arrayJobEnabled,
+              value: config?.isArrayJob,
+              onChange: onChange("isArrayJob"),
+            }}
+            showAlert={config?.isArrayJob === "Yes"}
+            alertType="info"
+            alertMsg="Please make sure you understand what is an array job."
+          />
+        ),
+        show: (config) =>
+          config?.service === "Lyra" && config?.jobType === "Batch",
+        selected: (config) =>
+          isValidChoice(arrayJobEnabled, config?.isArrayJob),
+      };
+    },
+    arrayJobConfig: () => {
+      return {
+        element: (key, selected) => (
+          <ConfigMultipleNumbers
+            key={key}
+            title="Array Job Config"
+            description="Array job Settings"
+            selected={selected}
+            value={config?.arrayJobConfig}
+            onChange={onChange("arrayJobConfig")}
+            inputProps={{
+              min: 1,
```

Comment:
The minimum for the first index should be 0 not 1, so this might need re-designing

Reply:
Updated